packages/combobox/src/Combobox.tsx

```diff
@@ -1137,6 +1137,7 @@ export function Combobox<M extends boolean>({
    *
    */
 
+  // TODO: Replace this with `useBackdropClick`
```

Comment:
Should we add this in now since the `useBackdropClick` PR has gone in now?

Reply:
I feel like this should be in a separate body of work